packages/format/src/index.ts

```diff
@@ -1,6 +1,62 @@
-import { InspectOptions } from 'util'
+// Some codes are derived from https://github.com/nodejs/node/blob/v18.7.0/lib/util.js
+
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
I added the copyright but not sure if it's required. I thought It would be nice to show respect anyway

Reply:
added README.md and moved copyright reference there 🙂